contracts/mainnet/connectors/liquity/main.sol

```diff
@@ -238,28 +245,27 @@ abstract contract LiquityResolver is Events, Helpers {
             lowerHint
         );
         
-        // Allow other spells to use the withdrawn collateral
-        setUint(setWithdrawId, withdrawAmount);
+        setUint(setIds[0], depositAmount);
+        setUint(setIds[1], withdrawAmount);
+        setUint(setIds[2], borrowAmount);
+        setUint(setIds[3], repayAmount);
```

Comment:
Should these setId values use the final value contained inside `adjustTrove.*` struct? Otherwise they might not be the same values that were sent to Liquity

Reply:
Ooh! Sorry. My bad. We can update this.

Reply:
Done https://github.com/liquity/dsa-connectors/pull/2/commits/027095603ce7b8e37e0d31f0660834bfd0a91c86